docs/troubleshooting.md

```diff
@@ -46,6 +48,41 @@ This is typically encountered when you attempt to install BKPR with a DNS zone (
 
 ![Azure SP Conflict](images/azure-sp-conflict.png)
 
+## Troubleshooting BKPR ingress
+
+### Let's Encrypt
+
+BKPR deploys [`cert-manager`](#components.md/cert-manager) Kubernetes add-on to automate the management and issuance of TLS certificates. `cert-manager` delegates on Let's Encrypt to retrieve valid X.509 certificates used to protect designated Kubernetes ingress resources with TLS encryption for HTTP traffic.
```

Comment:
the cert-manager Kubernetes add-on

Reword: `cert-manager` uses Let's Encrypt to retrieve valid X.509 certificates that protect designated Kubernetes Ingress resources with TLS encryption for HTTP traffic.

HTTP traffic -> do you mean HTTPS?